pixeltable/metadata/schema.py

```diff
@@ -224,7 +235,7 @@ class Table(Base):
     lock_dummy: orm.Mapped[int] = orm.mapped_column(BigInteger, nullable=True)
 
 
-@dataclasses.dataclass(frozen=True)
+@dataclasses.dataclass
```

Comment:
I substantially prefer keeping these as frozen dataclasses; if that's controversial then we should discuss.

Reply:
TableMd and TableSchemaVersionMd are not frozen.

Reply:
I'd argue they should be and we should make that change at some point (I'd be happy to do it).

Reverting TableVersionMd to a non-frozen data class is a step backwards IMO.